### ISSUE TITLE
Add instructions about local certificate usage / certification authority trust

### ISSUE THREAD

**Issue by ErikPelli** [elazarl/goproxy#560]
There are a lot of issues (like https://github.com/elazarl/goproxy/issues/142 and https://github.com/elazarl/goproxy/issues/529) where the users complain that the proxy is not accepted by their browser because their certificate authority is not among the trusted ones.
We should add some sections in the documentation where we explain how to do that.
Related to https://github.com/elazarl/goproxy/issues/222.

Basically, some instructions about how to add the existing goproxy certificate to the trusted ones by the browser or to generate a new valid one and trust it (and how to use it in goproxy, citing the already existing custom CA example).

You can add all the instructions to the README of the custom CA example.

**Comment by ErikPelli** [comment]
https://github.com/elazarl/goproxy/issues/417

**Comment by ErikPelli** [comment]
This issue is open for anyone
